### DISCUSSION TITLE
Image and audio observation type

### DISCUSSION THREAD

**Discussion by fpanty** [FraunhoferIOSB/FROST-Server#1315] [Q&A] [answered]
I was wondering how  FROST (SensorThingsApi) can be used with image/video/audio observation results. In particular I have a multispettral camera and a microphone mounted on a robot. The sensors  produce  respectively  a set of images and a wav file  on each oservation. 
Is the OM_CategoryObservation the right observation type with the result content (i.e. images or wav file) in a form of url pointing to an external web server where the image, audio files have been stored?

**Comment by hylkevds** [comment, marked as answer]
Yes, generally large binary results are put in an external media server, and the Observation result contains a link to the data. The observation type OM_CategoryObservation is for cases where there is a set of valid values (an enum).  In this case the most appropriate type would simply be OM_Observation.

The STA-Plus best-practice also deals with image data, and analysis results of this data: https://docs.ogc.org/bp/21-068.pdf

**Reply by fpanty** [reply]
Many thanks hylkevds!
